Firestore/core/test/firebase/firestore/util/path_test.cc

```diff
@@ -38,7 +38,7 @@ namespace util {
 //
 //   brew install llvm
 //   llvm=$(brew --prefix)/opt/llvm
-//   $llvm/bin/clang++ -I$llvm/include -I$llvm/include/c++/v1 -L$llvm/lib \
+//   $llvm/bin/clang++ -I$llvm/include -I$llvm/include/c++/v1 -L$llvm/lib
```

Comment:
not really sure why this '\' was deleted but I don't care too much...

Reply:
Trailing backslashes in a single-line comment are an error with gcc (but not clang). This backslash was breaking the build on Linux.